tests/unit/test_brute_force_sampling.py

```diff
@@ -0,0 +1,61 @@
+from unittest import mock
+import pytest
+
+from snowshu.samplings.samplings import BruteForceSampling
+
+
+@pytest.fixture()
+def mock_args():
+    mock_rel=mock.MagicMock()
+    mock_source_adapter=mock.MagicMock()
+    yield mock_rel,mock_source_adapter
+
+
+ONE_BILLION_ROWS=1e9
+ONE_HUNDRED_THOUSAND_ROWS=1e5
+def test_brute_force_sampling_stock(mock_args):
+    mock_args[0].population_size=ONE_BILLION_ROWS
+    brute_force=BruteForceSampling()
+    brute_force.prepare(*mock_args)
+
+    assert brute_force.sample_method.rows == 1000000
+
+def test_brute_force_sampling_fine(mock_args):
+    # GIVEN: mock_args - mock for snowshu.core.models.relation.Relation
+    #        2% rows to be sampled of ONE_HUNDRED_THOUSAND_ROWS
+    #        the min_sample_size is 3000
+    #        expectation - 1000000 rows to be sampled
```

Comment:
maybe it's because i'm new to snowshu, but i'd add a comment that  this 1M of rows as expected result is a consequence of snowshu configs.py configuration for better clarity and easier understanding